### PR TITLE
EVW3226 @ UPC support

### PR DESCRIPTION
**Pull request by carbolymer**
I've added support for EVW3226 router with customizations made by UPC. I had to move some common logic to other methods, so please check if I didn't break support of other routers.

### CODE REVIEW

File: pyubee/__init__.py [hunk 1]
```diff
@@ -62,6 +149,24 @@
             r'<td>([0-9a-fA-F]{12})</td>'  # mac address
             r'<td>(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})</td>'  # ip address
         ),
+        'authenticator': DefaultAuthenticator
+    },
+    'EVW3226@UPC': {
```
Comment (carbolymer) [inline pyubee/__init__.py:154]: Autodiscovery does not work for it (no model data without logging in), so I chose this name, which has to be typed manually as a script argument.

Comment (StevenLooman) [reply pyubee/__init__.py:154]: Does your device not have a /RootDevice.xml file? Perhaps you need to enable UPnP support on your device first?

**Comment by StevenLooman** [comment]
You'll probably have to rebase your PR, please do so. Also, please change the target branch to mzdrale:dev, version 0.6.dev0 (currently in the dev branch.)
Also, is there no way to make auto detection work?

Other than that, works for me/looks good to me.

@mzdrale What about you?

File: pyubee/__init__.py (hunk 1 above)
Comment (carbolymer) [reply pyubee/__init__.py:154]: It does not have it and I have UPnP enabled. It appears that the firmware was heavily modified by UPC - but they still left vulnerabilities lol. Router pages available without logging in do not contain model information.

**Comment by carbolymer** [comment]
> You'll probably have to rebase your PR, please do so. Also, please change the target branch to mzdrale:dev, version 0.6.dev0 (currently in the dev branch.)

~~Ok, will do after input from @mzdrale .~~ Done.

> Also, is there no way to make auto detection work?

Not that I am aware of. I've responded here: https://github.com/mzdrale/pyubee/pull/8#discussion_r275716085

**Comment by StevenLooman** [comment]
Ok then. Looks good to me. @mzdrale please review and merge if ok.

**Comment by mzdrale** [comment]
Sorry for the late response guys.
I wish there was a way to autodetect model, but it is what it is.
Looks good to me. Thanks @carbolymer for the PR and @StevenLooman for the review.

**Pull request merged**